### PR TITLE
CSV version of Transition Output

### PR DESCRIPTION
**Pull request by mattford63**
https://www.pivotaltracker.com/story/show/164468840

### CODE REVIEW

File: src/clj/witan/send/model/output.clj [hunk 1]
```diff
@@ -85,8 +85,24 @@
...
+           rows (sort (remove empty?
+                              (for [[[cy ay need-setting-source need-setting-destination] transitions] projections]
+                                (if (> (m/round0 (/ transitions simulations)) 0)
+                                  (into [] (flatten [cy
+                                                     ay
+                                                     (parse-need-setting need-setting-source)
+                                                     (inc ay)
+                                                     (parse-need-setting need-setting-destination)
+                                                     (m/round0 (/ transitions simulations))]))))))]
+       (csv/write-csv writer (into [headers] rows))))))
```
Comment (seb231) [inline src/clj/witan/send/model/output.clj]: Do we need the original one of these still? the .edn version?

Comment (mattford63) [reply src/clj/witan/send/model/output.clj]: Can't hurt to have it.  Will make a handy comparison if we wanted to test.  Neither are completely accurate at the moment but those fixes are to come.

Comment (seb231) [reply src/clj/witan/send/model/output.clj]: Not accurate, in that they don't capture everything?

Comment (mattford63) [reply src/clj/witan/send/model/output.clj]: And too much!  And oddly.

Comment (mattford63) [reply src/clj/witan/send/model/output.clj]: Should all be in the blogs/other big PR.

Comment (seb231) [reply src/clj/witan/send/model/output.clj]: okay. I'm not sure we need both, but it doesn't really hurt. I don't like however that the same fn does two very different things if you use 3 or 4 args. Could you separate them and call them different things please?

**Pull request merged**